views/albums.hbs

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
Because we already create a layout its not necessary to add again all the html,body and meta tags because our layout is already handle that for us 😎